src/controllers/index.ts

```diff
@@ -1,9 +1,12 @@
 import { Request, Response } from "express";
 
 /**
- * GET /
+ * GET 
  */
 export const index = async (_: Request, res: Response): Promise<void> => {
```

Comment:
We can remove this file now that we have a real route; it was just there for demonstration purposes.